src/templates/PoolManager/ActivityCard/TokenInfo/styles.ts

```diff
@@ -0,0 +1,31 @@
+import styled, { css } from 'styled-components'
+
+export const TokenInfo = styled.div`
+  ${() => css`
+    display: flex;
+    align-items: center;
+    gap: 0.8rem;
+
+    min-width: 14rem;
+  `}
+`
```

Comment:
You're not using any variable from `styled-components`, this arrow function is unnecessary
```suggestion
export const TokenInfo = styled.div`
  display: flex;
  align-items: center;
  gap: 0.8rem;

  min-width: 14rem;
`
```